README.md

```diff
@@ -53,4 +84,175 @@ To generate and access the API documentation on backend, follow these steps:
 
    You will see a page similar to the one shown below:
 
-![Api documentation](api-documentation.png)
+![Swagger Documentation]
```

Comment:
This line can be removed; unless there was a link that's missing here? Or if you mean to make it. a header, then you'll want:

```suggestion
#### Swagger Documentation
```